draft-ietf-rtcweb-mdns-ice-candidates.md

```diff
@@ -167,6 +167,12 @@ For any remote ICE candidate received by the ICE agent, the following procedure
 
 4. Otherwise, ignore the candidate.
 
+An ICE agent may add additional restrictions to resolve the ICE candidate using
```

Comment:
I thought about this some more and I think this is already a problem for raw IP addresses. It is covered in the existing rtcweb security docs, e.g., https://tools.ietf.org/html/draft-ietf-rtcweb-security-10#section-4.2.

My suggestion would simply be to add a note like what you have to the security considerations section, and simply mention that ICE already provides protection as noted in the reference above.

Reply:
Agreed for raw IP addresses, mDNS just makes the attack potentially faster.